tests/ArmTemplates.Tests/Extractor/Scenarios/ApiManagementServiceExtractorTests.cs

```diff
@@ -0,0 +1,105 @@
+﻿// --------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+// --------------------------------------------------------------------------
+
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Commands.Executors;
+using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Common.Constants;
+using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Common.FileHandlers;
+using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Common.Templates.ApiManagementService;
+using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Common.Templates.Builders;
+using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Extractor.EntityExtractors;
+using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Extractor.Models;
+using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Tests.Extractor.Abstractions;
+using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Tests.Moqs.ApiClients;
+using Microsoft.Azure.Management.ApiManagement.ArmTemplates.Common.Extensions;
+using Xunit;
+
+namespace Microsoft.Azure.Management.ApiManagement.ArmTemplates.Tests.Extractor.Scenarios
+{
+    [Trait("Category", "ApiManagementService Extraction")]
+    public class ApiManagementServiceExtractorTests : ExtractorMockerWithOutputTestsBase
+    {
+        public ApiManagementServiceExtractorTests() : base("api-management-service")
+        {
+        }
+
+        [Fact]
+        public async Task GenerateApiManagementService_ProperlyLaysTheInformation()
+        {
+            // arrange
+            var currentTestDirectory = Path.Combine(this.OutputDirectory, nameof(GenerateApiManagementService_ProperlyLaysTheInformation));
+
```

Comment:
let's group rows somehow. Every row is splitted with an empty line, this is hard to read IMO

Reply:
Agree, removed new liners in both tests